### DISCUSSION TITLE
Danish Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#345] [Language Leaders]
@fadamsen, @rhl2401 - moving discussion about Danish language leader(s) here.

**Comment by fadamsen** [comment]
I want to be Danish Language Leader because I believe I have the desire and skills necessary to do it well.

I have a strong command of the Danish language, I can obsess about grammar _when I want to_, I know my way around internationalization and localization (though that doesn't matter as much for intents), and I really like to get things done.

My experience with Home Assistant goes back about three years, so I'm no stranger to the ecosystem, and I'm familiar with the challenges of using Siri for controlling my home, as the quality you would normally associate with Apple just...isn't there, in my opinion. I hope that Home Assistant, with a strong focus on voice, can change that, and I want to be a part of it.

For now, I'm working on sentences and tests, trying to lay a foundation that we can build on in the years to come. I look forward to working with you all on this!

**Comment by synesthesiam** [comment]
Sounds great @fadamsen! I've added you as the Danish language leader :slightly_smiling_face:

**Comment by MTrab** [comment]
I'd like to apply as well.
I'm one of the co-admins of the Danish Home Assistant group on Facebook and author of several custom_integrations as well.

I've been onboard the HA system from a very early state and I know of several users that enjoys my high knowledge level when it comes to the more deeper uses of Home Assistant.

**Comment by synesthesiam** [comment]
Great, thanks @MTrab! We're asking people to please submit 2 PRs to become a language leader: one per HA domain, translating errors, etc. :+1:

**Reply by MTrab** [reply]
PRs to the language or PRs to the HA community? Got several 100 PRs done

**Reply by synesthesiam** [reply]
I was meaning language, but it looks like from your history you clearly know your way around Github :laughing: 
Let me just add you here.

**Reply by MTrab** [reply]
Indeed I do :D Was actually about to go crazy in the translations during the weekend - but let's see how many we are left with when I get off job tomorrow ;)

And thx.

**Reply by fadamsen** [reply]
Hi @MTrab! I’m working on covers and expect to finish those tonight, but I won’t get around to climate, so there’ll definitely be something to do! 😊

**Reply by MTrab** [reply]
Cool :) You could hit me up on Discord if you like, then we can continue on there.

**Reply by fadamsen** [reply]
I'll do that tomorrow!

**Comment by synesthesiam** [comment]
@fadamsen @MTrab when you get a chance, can you please look into translating the sentences for `HassGetState`?

**Reply by fadamsen** [reply]
I have been a bit overwhelmed with work for a while, I'm working to catch up now.

**Reply by synesthesiam** [reply]
No worries, I appreciate it :slightly_smiling_face:

**Reply by MTrab** [reply]
Same for me - way too much work and way too many chores for some time now.

**Comment by fadamsen** [comment]
Hi @synesthesiam and @MTrab!

I’m stepping down as Danish language leader. I haven’t been able to contribute much, if at all, in the past year, and I sadly don’t see this changing anytime soon.

I hope to be able to contribute more in the future, but for now, this is it for me.
